server/middleware/searchHandler.js

```diff
@@ -70,6 +68,19 @@ async function getMemberExternalKey(memberUuid) {
     return response.data.externalKey;
 }
 
+const returnSearchWorkstackColumns = (workstackTypeColumns, request) => {
+    let searchWorkstackColumns = workstackTypeColumns.find(
+        item => item.workstackType === `${request.caseType}_SEARCH_RESULTS`
+    );
+
+    if (searchWorkstackColumns === undefined) {
+        searchWorkstackColumns = workstackTypeColumns.find(
+            item => item.workstackType === 'DEFAULT'
+        );
+    }
+    return searchWorkstackColumns;
+};
+
 module.exports = {
-    handleSearch
+    handleSearch, returnSearchWorkstackColumns
```

Comment:
Split this over two lines so that the Git diff looks nicer if we edit one in future.

Reply:
I have moved this method elsewhere